wpk/run.sh

```diff
@@ -28,17 +29,18 @@ help() {
     echo "Usage: ${0} [OPTIONS]"
     echo "It is required to use -k or --aws-wpk-key, --aws-wpk-cert parameters"
     echo
-    echo "    -b,   --branch <branch>      [Required] Select Git branch or tag e.g. master"
-    echo "    -o,   --output <name>        [Required] Name to the output package."
-    echo "    -pn,  --package-name <name>  [Required for windows] Package name to pack on wpk."
-    echo "    -r,   --revision <rev>       [Optional] Revision of the package. By default: 1."
-    echo "    -p,   --path <path>          [Optional] Installation path for the package. By default: /var."
-    echo "    -j,   --jobs <number>        [Optional] Number of parallel jobs when compiling."
-    echo "    -c,   --checksum             [Optional] Whether Generate checksum or not."
-    echo "    --aws-wpk-key                [Optional] AWS Secrets manager Name/ARN to get WPK private key."
-    echo "    --aws-wpk-cert               [Optional] AWS secrets manager Name/ARN to get WPK certificate."
-    echo "    --aws-wpk-key-region         [Optional] AWS Region where secrets are stored."
-    echo "    -h,   --help                 Show this help."
+    echo "    -b,   --branch <branch>               [Required] Select Git branch or tag e.g. master"
+    echo "    -o,   --output <name>                 [Required] Name to the output package."
+    echo "    -pnw,  --package-name-windows <name>  [Required for windows] Package name to pack on wpk, incompatible with -pnm."
+    echo "    -pnm,  --package-name-macos <name>    [Required for macos] Package name to pack on wpk, incompatible with -pnw."
```

Comment:
Duplicate options isn't needed.

Reply:
Done.
I removed the duplicate -pnm / -pnw options. I used the file extension to recognize the package type.